src/system/apps/base/installable_app.py

```diff
@@ -21,7 +21,7 @@
 class InstallableApp(BaseModel, ABC):
 
     def __init__(self, display_name, repo_name, service_file_name, data_dir_name, port, min_support_version,
-                 description='', gateway_access=False, url_prefix='', version=''):
+                 description='', gateway_access=False, url_prefix='', version='', need_wires_plat=False):
```

Comment:
`need_wires_plat=True` by default and only send value `need_wires_plat=False` for `wires-plat` app (leave others as is was before).